hw/bootrom/cheshire_bootrom.ld

```diff
@@ -9,7 +9,7 @@
 INCLUDE common.ldh
 
 SECTIONS {
-  __stack_pointer$  = ORIGIN(memisl) + LENGTH(memisl) - 8;
+  __stack_pointer$  = __stack_end - 8;
```

Comment:
Why `__stack_end`? The stack should start at the end of the `L2 Memory`. I would name this more intuitively.

Reply:
ok